front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/tag/DefaultEntityTagsDAO.java

```diff
@@ -51,17 +52,6 @@ private EntityTags upsert(String id, EntityTags tag) {
   }
 
   @Override
-  public Collection<EntityTags> all() {
```

Comment:
This is definitely better for OSS folks (not that anyone is using it yet, but still).